### ISSUE TITLE
The LAION-ART (Filtered) 600K Dataset

### ISSUE THREAD

**Issue by XIRZC** [modelscope/scepter#9]
Hello,

Thanks for you great work!

In the published paper, it was mentioned, "In controllable image synthesis tasks, for each type of conditional data, we employed a refined version of the LAION artistic dataset [37], comprising approximately 600k images. This refined dataset excludes duplicates, low-resolution images, content with potential harm, and those of subpar quality."

We are interested in accessing the **filtering program or filtered metadata**, such as links and captions (like JSON or Parquet format). This would greatly aid our research efforts.

Looking forward to your response.

**Comment by jiangzeyinzi** [comment]
As described in our work, we applied several filters to the LAION data, including checks for watermarks (pwatermark), unsafe content (punsafe), aesthetic scores, and resolution quality. Additionally, we utilized classifiers that were trained through manual annotation. Due to organizational restrictions, we may not be able to make this subset (600k) list of LAION-ART available. BTW, we discovered that an increased volume of data typically enhances the quality of generated results.
